### PR TITLE
Anja xxx ml toolkit

### PR DESCRIPTION
**Pull request by AnjaNeumann**

### CODE REVIEW

File: docs/ml-toolkit/example.md [hunk 1]
```diff
@@ -38,6 +46,13 @@ a spiky model response which has mainly two reasons:
...
+<p align="center">
+<img src="../img/train_test_assignment_rnd.png" width="70%" height="70%">
+</p>
+<p align = "center"><i>
+Random Assignment of Train/Test Split</i>
+</p>
+
 ### Block Sampling
 
 It is therefore mandatory to enable machine learning that respects the basic
```
Comment (AliceBalfanz) [inline docs/ml-toolkit/example.md:58]: I don't understand why the word "therefore" is used.

**Review by AliceBalfanz — CHANGES_REQUESTED** [review]
Please see comments and also the abbreviation ARDC is explained no where. Please add that :)

File: docs/ml-toolkit/example.md (hunk 1 above)
Comment (AnjaNeumann) [reply docs/ml-toolkit/example.md:58]: It assigns to the problems mentioned before (see Random Sampling)

**Comment by AnjaNeumann** [comment]
done

**Pull request merged**